requirements-dev.txt

```diff
@@ -1,2 +1,2 @@
 behave==1.2.5
-Markdown==2.5.2  # FIXME: 2.6.8 breaks tests
+Markdown==2.6.8
```

Comment:
I did try it without that change, an there's one extra failing test under this--it's not the widespread problem we saw before, so I think I'll leave it for now.